src/command_line_parser.cc

```diff
@@ -1345,7 +1350,8 @@ TritonParser::Parse(int argc, char** argv)
           lparams.http_address_ = optarg;
           break;
         case OPTION_HTTP_HEADER_FORWARD_PATTERN:
-          lparams.http_forward_header_pattern_ = optarg;
+          lparams.http_forward_header_pattern_ =
+              std::move(case_insensitive_prefix + optarg);
```

Comment:
We should also update the docs on the header forward pattern feature to specifically mention that it will match case insensitively.

Reply:
https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_parameters.md#forwarding-httpgrpc-headers-as-parameters